ta4j-core/src/test/java/org/ta4j/core/bars/AmountBarBuilderTest.java

```diff
@@ -30,14 +30,15 @@
 import java.time.Instant;
 
 import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
 import org.ta4j.core.BaseBarSeriesBuilder;
 import org.ta4j.core.num.DecimalNumFactory;
 
 public class AmountBarBuilderTest {
 
-    @Test(expected = IllegalArgumentException.class)
-    @SuppressWarnings("unused")
+    @Test
     public void createBarsWithSetAmountByVolume() {
```

Comment:
Aaaand JUnit5 supports package private test methods and classes, we can remove public modifiers.